src/lib/strapi.ts

```diff
@@ -56,6 +57,7 @@ export type ProfileData = {
     publishedAt: string | null
     avatar?: StrapiData<AvatarData>
     gravatarURL: string | null
+    questionSubscriptions: QuestionData[]
```

Comment:
```suggestion
    questionSubscriptions: StrapiData<QuestionData[]>
```

Reply:
Believe there are some type differences that need to be sorted between questions and users, so updating to this still throws errors

Reply:
Yup I was mistaken here, for whatever inexplicable reason, the format that gets returned from the `/api/users/me` endpoint doesn't match with the normal API endpoints 😠 

You're fine to revert to the original type, or if not I'll go through and fix this in a separate PR